igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Can be simplified to:</p><p> </p><p><code></code><code>python</p><p>data_dir = Path('data')</p><p>data_dir.mkdir(exists_ok=True)</p><p>...</p><p></code><code></code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>